libs/designsystem/src/lib/components/card/card-header/card-header.component.scss

```
@@ -23,3 +33,32 @@ h4 {
   margin: 0;
   font-weight: font-weight('normal');
 }
+
+// Flagged card headers
+
+// key = level, value = color-name
+$notification-levels: (
+  'info': 'semi-light',
+);
+
+@each $color-name, $color-value in $notification-colors {
+  $notification-levels: map-merge(
+    (
+      $color-name: $color-name,
+    ),
+    $notification-levels
+  );
+}
+
+@each $level, $color-name in $notification-levels {
+  :host(.#{$level}) {
+    --kirby-card-header-background-color: #{get-color($color-name)};
+    --kirby-card-header-color: #{get-color($color-name + '-contrast')};
+    --kirby-card-header-padding: #{size('xxxxs')} #{size('xxs')};
```

Consider if these variables a part of the public API - if not, they should be renamed:
```suggestion
    --kirby_internal_card-header-background-color: #{get-color($color-name)};
    --kirby_internal_card-header-color: #{get-color($color-name + '-contrast')};
    --kirby_internal_card-header-padding: #{size('xxxxs')} #{size('xxs')};
```